src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = number => {
+  const myArray = Array.from(number.toString()).map(Number);
```

Comment:
nice use of from an then mapping to a number using the constructor function. Is it clear why you can just put Number in as an argument for your map?

Reply:
Hey Romy, I feel like it was clear when I did it why I couldn't put number in but it isn't now I've come back to it haha. Are you able to explain please?

Reply:
So you can use a shorthand when you are passing the first argument of a function to another function. For example the first argument of a map when you have this array is each element in the array `const array = ["string", "string2"];` You can write your map as `array.map(element => Number(element));` The shorthand for anything where you don't need to mutate or manipulate the argument you are passing in is you can just write the function name and then the first argument will be implicitly passed into the function inside the map. So `array.map(element => Number(element))` is the explicit way to pass the element into the Number function and then `array.map(Number)` is the implicit way of writing it. Similar to when you have a return of an arrow function `() => { return result }` is the explicit way of returning it but the implicit way of writing it is `() => result`.

Reply:
Hope that makes sense.

Reply:
@anywhereiromy thank you, that makes sense again :)